docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -154,6 +155,44 @@ \section{Failure Mode and Effect Analysis}
       Error messages and logging & a. Allow administrator to adjust
       permissions & SR-AR1 \newline SR-AR4 & H3-2 \\
       \hline
+      Accept Contractor Document Upload & Document is valid but is uploaded in
+      the wrong location & Poor discoverability and organization &
+      a. Human-computer interface failure &User detects &
+      a. Alert administrator to adjust & FR1 & H4-1 \\
+      \cline{2-8}
+      & Document is malicious and should not be in the system & Malware
+      infection &a. Malicious user gained access to system & Error messages and
+      logging & a. Remove file from system and notify administrator & SR-AR1
+      & H4-2 \\
+      \cline{2-8}
+      & File chosen for upload is corrupted and not able to be opened &
+      User fails to open file &a. Failure in document chosen or upload process&
+      Error messages and logging & a. Remove file from system and
```

Comment:
remove logging

Reply:
who removes file

Reply:
removed logging, specified that the application removes the file